app/src/main/java/com/nmakademija/nmaakademija/MainActivity.java

```diff
@@ -27,28 +28,34 @@ protected void onCreate(Bundle savedInstanceState) {
         bottomNavigation = (BottomNavigation) findViewById(R.id.bottom_navigation);
 
         bottomNavigation.setOnMenuItemClickListener(this);
-        setCurrentFragment(0);
+        setCurrentFragment(0, getSupportActionBar());
     }
 
-    void setCurrentFragment(int position) {
+    void setCurrentFragment(int position, android.support.v7.app.ActionBar bar) {
         Fragment fragment;
         FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
 
 
         switch (position) {
-            case 0:
-                fragment = NewsFragment.getInstance();
-                break;
             case 1:
-                fragment = ScheduleFragment.getInstance();
+                fragment = UsersFragment.getInstance();
+                bar.setBackgroundDrawable(new ColorDrawable(getResources().getColor(R.color.bottomNavigationItem2)));
+                bar.setSubtitle(getResources().getString(R.string.users));
                 break;
             case 2:
                 fragment = TimeUntilSessionFragment.getInstance();
+                bar.setBackgroundDrawable(new ColorDrawable(getResources().getColor(R.color.bottomNavigationItem3)));
+                bar.setSubtitle(getResources().getString(R.string.timer));
+                break;
+            case 3:
+                fragment = ScheduleFragment.getInstance();
+                bar.setBackgroundDrawable(new ColorDrawable(getResources().getColor(R.color.bottomNavigationItem4)));
+                bar.setSubtitle(getResources().getString(R.string.schedule));
```

Comment:
Create two variables before switch: `color` and `subTitle`. Set these inside switch. After switch call:
```java
bar.setBackgroundDrawable(new ColorDrawable(getResources().getColor(color)));
bar.setSubtitle(getResources().getString(subTitle));
```